test_requirements.txt

```diff
@@ -7,6 +7,11 @@
 pytest>=3.0.0; python_version>="3.4"
 pytest>=3.0,<3.3; python_version<"3.4"
 
+# isort is an indirect requirement of pylint. isort 4.3 dropped support
+# for Python 3.3, so we need to peg it to an earlier version to avoid
+# breakages.
+isort==4.2.15; python_version<"3.4"
```

Comment:
We're doing a lot of things to support python 3.3, which is no longer even supported by pytest and a lot of other tools. I think we should drop 3.3 support altogether

Reply:
Okay. I'll drop 3.3 support in this PR.